### PR TITLE
Misc Documention Fixes

### PR DESCRIPTION
**Pull request by rglazer2**
Addresses #4615

### CODE REVIEW

File: doc/src/docs/InterfaceDeveloper/InterfaceDeveloper.md [hunk 1]
```diff
@@ -1108,7 +1108,7 @@ Licensing
 
 In order to make efficient distribution of your interface, you should consider licensing EnergyPlus.  First, you must license/register to use EnergyPlus.  Since we are distributing EnergyPlus via the World Wide Web, at no charge, it will be easy to get your hands on a copy for testing out your interface.  However, you may wish to understand more of the internals and, to make a complete distribution package, will need to at least execute a distribution license.
 
-http://www.eere.energy.gov/buildings/energyplus/energyplus_licensing.html contains the details on licensing EnergyPlus for commercial or non-commercial use.
+http://apps1.eere.energy.gov/buildings/energyplus/energyplus_licensing.cfm contains the details on licensing EnergyPlus for commercial or non-commercial use.
```
Comment (mjwitte) [inline doc/src/docs/InterfaceDeveloper/InterfaceDeveloper.md]: @rglazer2 Things are moving.  Please use this URL instead
https://energyplus.net/licensing

**Comment by Myoldmopar** [comment]
@rglazer2 I wouldn't have anything to add on to the comment @mjwitte noted above about using the updated licensing link.  If you can update that in your branch, I'll merge these changes.

On a side note, @mjwitte, I know you keep lists of doc issues that you encounter.  Have you done any "automated" passes over the docs looking for old links?  If not, I'll do that.

**Comment by mjwitte** [comment]
@Myoldmopar No, I have not done any mass link check.

**Comment by Myoldmopar** [comment]
@mjwitte OK, should be easy enough to script that.  I'll add it on my list now.

**Comment by Myoldmopar** [comment]
@rglazer2 Thanks for updating the link.  If you can do a quick update from our latest develop, it would be great as the dashboard should go back to all green and I'll merge it in.

**Comment by rglazer2** [comment]
@Myoldmopar Should be updated with the most recent develop now.

**Pull request merged**